CHANGES.rst

```diff
@@ -1,10 +1,11 @@
 0.20.0 (unreleased)
 -------------------
 
-other
-^^^^^
+- Improve documentation (part 1) [#483]
```

Comment:
I'm not sure about whether documentation changes are to be noted in the CHANGES.rst. In this case there are errors in the documentation, so that may be worth noting. Perhaps Nadia can clarify for us.